.github/labeler.yml

```diff
@@ -32,6 +32,17 @@ controller mappings:
       - any-glob-to-any-file:
           - res/controllers/**
 
+developer experience:
+  - changed-files:
+      - any-glob-to-any-file:
+          - .github/ISSUE_TEMPLATE/**
+          - .github/labeler.yml
+          - tools/**
```

Comment:
Not sure if `tools` is relevant for this tags - there is a lot of tools in here which aren't related to developer experience IMO.
I would also argue that labeler is not related to developer experience either

Reply:
I'd say that labeler is related to `development processes`, and thus fits the `developer experience` tag.

re: `tools/`: AFAICT the labeler is only applied initially to new pull requests, and the labels added using these heuristics can be removed manually if desired, so a bit of over- or underlabeling is better than none at all.

But I'm also fine with removing both `.github/labeler.yml` and `tools/`, every way of labeling will be some kind of "best effort" anyway.

Reply:
Let's keep it like this for now, we can always remove it if we get too many false positive